zou/utils/movie.py

```diff
@@ -204,16 +191,46 @@ def add_empty_soundtrack(file_path):
         "-shortest",
         tmp_file_path,
     ]
+    logger.info(f"Launch ffmpeg with args: {' '.join(args)}")
     sp = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     out, error = sp.communicate()
 
     err = None
     if error:
         err = "\n".join(str(error).split("\\n"))
 
-    print("sp.returncode: {}".format(sp.returncode))
+    logger.info(f"add_empty_soundtrack.sp.returncode: {sp.returncode}")
     if sp.returncode == 0:
         shutil.copyfile(tmp_file_path, file_path)
+    else:
+        logger.error(f"Err in soundtrack: {err}")
+        logger.error(f"Err code: {sp.returncode}")
+        if try_count <= 1:
+            (width, height) = get_movie_size(file_path)
+            if height % 2 == 1:
+                height = height + 1
+            stream = ffmpeg.input(file_path)
+            stream = ffmpeg.output(
```

Comment:
Why don't you use the function you created before (ffmpeg_normalize)?

Reply:
Because I don't pass the arguments :
- `stream.audio`
- `r=fps,`
- b=b,`

This could be fixed in a future refactoring.